### PR TITLE
UI: use `KvPageHeader` consistently

### PR DESCRIPTION
**Pull request by hellobontempo**

### CODE REVIEW

File: ui/lib/kv/addon/components/kv-page-header.hbs [hunk 1]
```diff
@@ -29,15 +27,11 @@
 
 {{#if (or (has-block "toolbarFilters") (has-block "toolbarActions"))}}
   <Toolbar>
-    {{#if (has-block "toolbarFilters")}}
-      <ToolbarFilters>
-        {{yield to="toolbarFilters"}}
-      </ToolbarFilters>
-    {{/if}}
+    <ToolbarFilters>
```
Comment (Monkeychip) [inline ui/lib/kv/addon/components/kv-page-header.hbs:30]: I kind of remember an issue of styling showing the ToolbarFilters html (basically just a nav and class) for views that showed no filters. Just making sure we double checked there were no styling issues if toolbarFilters was empty.

File: ui/lib/kv/addon/components/page/list.hbs [hunk 2]
```diff
@@ -1,6 +1,6 @@
-<KvPageHeader @breadcrumbs={{@breadcrumbs}} @isEngineView={{true}} @pageTitle={{@backend}}>
+<KvPageHeader @breadcrumbs={{@breadcrumbs}} @mountName={{@backend}}>
   <:tabLinks>
-    <LinkTo @route={{@routeName}} data-test-secrets-tab={{@routeName}}>Secrets</LinkTo>
+    <LinkTo @route={{this.router.currentRoute.localName}} data-test-secrets-tab="Secrets">Secrets</LinkTo>
```
Comment (Monkeychip) [inline ui/lib/kv/addon/components/page/list.hbs:3]: I just discovered this router property the other day! I remember thinking, gosh I should use it more; and this is a great use case.

File: ui/lib/kv/addon/routes/secret/metadata/diff.js
Comment (Monkeychip) [inline ui/lib/kv/addon/routes/secret/metadata/diff.js:1]: nice clean up here. thank you.

File: ui/tests/integration/components/kv/kv-page-header-test.js [hunk 3]
```diff
@@ -38,12 +38,9 @@ module('Integration | Component | kv | kv-page-header', function (hooks) {
 
   test('it renders breadcrumbs', async function (assert) {
     assert.expect(4);
-    await render(
-      hbs`<KvPageHeader @breadcrumbs={{this.breadcrumbs}} @model={{this.model}} @pageTitle="Create new version"/>`,
-      {
-        owner: this.engine,
-      }
-    );
```
Comment (Monkeychip) [inline ui/tests/integration/components/kv/kv-page-header-test.js:46]: oh, good catch.

**Review by Monkeychip — APPROVED** [review]
Looks great! I did a quick check by checking out the branch. Thanks for this!
Edit: noticed when you navigate to a directory, the breadcrumbs don't work as expected. I can help problem solve this tmrw if you need an extra set of eyes. That nested breadcrumbs link gets 🙃

**Review by Monkeychip — CHANGES_REQUESTED** [review]
Edit: noticed when you navigate to a directory, the breadcrumbs don't work as expected. I can help problem solve this tmrw if you need an extra set of eyes. That nested breadcrumbs link gets 🙃

**Comment by hellobontempo** [comment]
> Edit: noticed when you navigate to a directory, the breadcrumbs don't work as expected. I can help problem solve this tmrw if you need an extra set of eyes. That nested breadcrumbs link gets 🙃

Yeah! My test caught this! Debugging now :)

**Pull request merged**